src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOFalcon.kt

```diff
@@ -233,7 +376,7 @@ class SwerveModuleIOFalcon(
   }
 
   override fun zeroDrive() {
```

Comment:
clear the queue when you zero, because if queue is not empty, that stuff will all still go into pose est